app/persistence/fabric/CRUDService.ts

```diff
@@ -540,6 +564,22 @@ export class CRUDService {
 	}
 	// Orderer BE-303
 
+	/**
+	 *
+	 *
+	 * @param {*} blockHeight
+	 * @returns
+	 * @memberof CRUDService
+	 */
+	async isBlockAvailableInDB(channel_genesis_hash: string, blockHeight: any) {
+		const count: any = await this.sql.getRowsBySQlCase(
```

Comment:
Please give specific datatype instead of  'any'.

Reply:
Updated the type to 'number' replacing 'any'.